Add filtering information to processing docs (#20)
AlexsLemonade/scpca-docs: docs/processing_information.md

#### Filtering cells

> In addition to an unfiltered counts matrix, we provide a matrix filtered to only droplets that are likely to include cells, removing any cell barcodes that are likely to correspond to empty droplets.

Reviewer: The second clause seemed redundant to me, but including "cell barcodes" seems important, so what do you think of this?
```suggestion
In addition to an unfiltered counts matrix, we provide a matrix filtered to only cell barcodes from droplets that are likely to include true cells.
```

Author: I like it! Will go ahead and commit that change.
